app/reports/page-content.tsx

```diff
@@ -232,7 +250,9 @@ export const ReportsPageContent = () => {
         }
 
         if (appealed) {
-          queryParams.appealed = appealed
+          // If not specifically set to true but there is a value, we can default to false
+          // No value will pass undefined which will be ignored
+          queryParams.appealed = appealed === 'true'
         }
```

Comment:
I wonder if an unknown value would be better suited to be interpreted as "unset" rather than `false`, and some of the logic above seems to handle it that way.

Reply:
E.g. something like:
```js
        if (appealed === 'true') {
          queryParams.appealed = true
        } else if (appealed === 'false') { 
          queryParams.appealed = false
        }
```

Reply:
hmm... I think I see what you mean but params.get returns null and setting it to an "unknown" value feels odd and could only be needed if we ever give this more meaning in which case we'd need another touchpoint here anyways. so I'll leave it as is for now but happy to discuss more if you feel strongly.

Reply:
The main thing I was noticing is that in some logic (mostly the display logic) an unknown value was treated as `false` and in other logic it was treated as unset.  So it looked like a consistency issue, e.g. the UI might display something different than what the server was receiving.

Reply:
(My inclination is it ignore unknown data and keep the default behavior rather than interpret it to a specific value.)

Reply:
Trying to illustrate better!  I expect the current behavior is like this:
 - `?appealed=`
   - UI shows "Appeals"
   - 👍 Server doesn't filter based on appeal
 - `?appealed=true`
   - UI shows "Only Appeals"
   - 👍 Server services "only appeals"
 - `?appealed=false`
   - UI shows "No Appeals"
   - 👍 Server services "no appeals"
 - `?appealed=unknown`
   - UI shows "Appeals"
   - ⚠️ Server services "no appeals"

Reply:
makes sense. will get it wired up in the next PR! thanks